python/copr/v3/proxies/project.py

```diff
@@ -110,6 +110,7 @@ def add(self, ownername, projectname, chroots, description=None, instructions=No
             enabled together with this project repository.
         :param list packit_forge_projects_allowed: List of forge projects that
             will be allowed to build in the project via Packit
+        :param str storage: What storage should be set for new projects
```

Comment:
Please add here the comment about admin only action too

Reply:
Also I can see that in cli and frontend there are choices between pulp and BE. Can we somehow here check whether this is also storage enum or some typo nonsense?

Reply:
> Please add here the comment about admin only action too

Good idea, I will mention it.

> Can we somehow here check whether this is also storage enum or some typo nonsense?

That would be a precedent because so far we don't have validation for any fields in `python-copr`, only in CLI and then on frontend. Having validation there could be nice if the code could be shared between `python-copr` and `copr-cli` but I don't like the idea of duplicating the validation code on yet another place.

Reply:
> Please add here the comment about admin only action too

Done